### ISSUE TITLE
Is this correct that the same items is drop from candyshop

### ISSUE THREAD

**Issue by l4svl** [ValveSoftware/Dota2-Gameplay#19385]
### Description

The pudge arcana is droped twice to my account, i`ve already received it and candy shop is suggesting me to get another one

### Example Match ID (and possibly Timestamp)

307587001 id of account

### Screenshots

![photo_2024-06-14_04-31-19](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109246430/d49d905e-e315-4224-8ef8-55811a53bf3b)
![photo_2024-06-14_04-31-32](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109246430/3f6e59ba-585b-43b5-aa5d-008a1e79ee1f)
![photo_2024-06-14_04-31-35](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109246430/42f4d84e-baa4-4b75-8366-21d3bcd60b60)

**Comment by Paper-Nap** [comment]
Yes, any item can drop any number of times, getting an item on a roll does not remove it from the roll basket.
